python/tools/wlr-libpy/README.md

```diff
@@ -0,0 +1,48 @@
+# About
+
+Helper crate for linking to the pre-built libpython from Webassembly Language Runtimes.
+
+# Features
+
+## build
+
+This feature is intended for usage in `build.rs` scripts. It will download all needed pre-built static libraries for `wasm32-wasi` and configure the linker to use them.
+
+To use this feature add this to your Cargo.toml
+
+```toml
+[build-dependencies]
+wlr-libpy = { git = "https://github.com/vmware-labs/webassembly-language-runtimes.git", branch="rust-py-example", features = ["build"] }
```

Comment:
```suggestion
wlr-libpy = { path = "../../../wlr-libpy", features = ["build"] }
```

Reply:
Same as mentioned in the other comment. Let's keep the repo but remove the branch.